packages/components/src/custom-select-control/test/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { render, fireEvent } from '@testing-library/react';
+/**
+ * WordPress dependencies
+ */
+import { CustomSelectControl } from '@wordpress/components';
+
+describe( 'CustomSelectControl', () => {
+	const onKeyDown = jest.fn();
+
+	it( 'Captures the keypress event and does not let it propagate', () => {
+		const wrapper = (
+			<div
+				role="none"
```

Comment:
To stop eslint reporting an issue with the div having a `keyDown` listener but no role.

Reply:
Sounds good! Could you add this as an inline comment too?